### ISSUE TITLE
Project file unable to load due to zoom in.

### ISSUE THREAD

**Issue by SamuelCurrier** [Scirra/Construct-3-bugs#3533]
I zoomed into the layout too much and it crashes every time I load the layout. I am unsure how to reset the zoom since it doesn't let me view the layout anymore. I am also using experimental features.

Error report information
Type: unhandled exception
File: https://editor.construct.net/r171-3/main.js, line 63, col 26289
Message: Uncaught Error: texture exceeds maximum texture size
Stack: Error: texture exceeds maximum texture size at ǃp.ǃKv.ǃYrW.ǃYry (https://editor.construct.net/r171-3/main.js:63:26295) at ǃp.ǃKv.ǃgR.ǃUL (https://editor.construct.net/r171-3/main.js:63:58965) at window.ǃsH.ǃUH (https://editor.construct.net/r171-3/components/editors/layoutView/layoutView.js:1:8491) at window.ǃsH.ǃsX (https://editor.construct.net/r171-3/components/editors/layoutView/layoutView.js:1:7327) at ǃsh (https://editor.construct.net/r171-3/components/editors/layoutView/layoutView.js:1:1844)
Construct 3 version: r171.3
URL: https://editor.construct.net/
Date: Sun Dec 08 2019 09:13:53 GMT-0500 (Eastern Standard Time)
Uptime: 73.9 s

Platform information
Browser: Chrome
Browser version: 78.0.3904.108
Browser engine: Chromium
Browser architecture: 64-bit
Context: webapp
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 8
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36
C3 release: r171.3 (stable)
Language setting: en-US

WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Google Inc.
Renderer: ANGLE (Intel(R) UHD Graphics 620 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions: EXT_color_buffer_float, EXT_disjoint_timer_query_webgl2, EXT_float_blend, EXT_texture_filter_anisotropic, KHR_parallel_shader_compile, OES_texture_float_linear, WEBGL_compressed_texture_s3tc, WEBGL_compressed_texture_s3tc_srgb, WEBGL_debug_renderer_info, WEBGL_debug_shaders, WEBGL_lose_context

**Comment by Wackytoaster** [comment]
I tried zooming in as much as it would allow me but I can´t reproduce that. You should maybe try opening it in the latest beta and see if you can zoom out there (you can still revert back to the stable release later by editing the c3p but make a backup just in case)

If this doesn´t help you should probably attach the project (or send it to ashley via mail if you don´t want it public)

**Comment by SamuelCurrier** [comment]
Thank you for the response, I did fix the issue but I can't remember how.
It may have been because I had an instance of an object selected and then
zoomed in but I'm not sure.

**Comment by AshleyScirra** [comment]
Normally we can't investigate reports without a project as per the guidelines. However with a bit of guesswork I think I figured this out and assume it is the cause of the problem here. If you set a very large grid size in the layout view and then zoom in a long way, the size of the grid cell image could become extremely large, which could then cause this crash as it exceeds the maximum texture size. I've added a maximum size for the next release which avoids this problem.
